docs/encoders.md

```diff
@@ -39,3 +43,112 @@ That's it, now on every reboot the motor will turn in one direction until it fin
 
 * If you wish to scan for the index pulse in the other direction (if for example your axis usually starts close to a hard-stop), you can set a negative value in `<axis>.encoder.config.idx_search_speed`.
 * If your motor has problems reaching the index location due to the mechanical load, you can increase `<axis>.motor.config.calibration_current`.
+
+### Startup sequence notes
+
+The following are variables that MUST be set up for your encoder configuration. Your values will vary depending on your encoder:
+
+* `<axis>.encoder.config.cpr = 4000`
+* `<axis>.encoder.config.mode = ENCODER_MODE_INCREMENTAL`
+
+The following are examples of values that MAY impact the success of calibration. Your values will vary depending on your setup:
+* `<axis>.motor.config.motor_type = MOTOR_TYPE_HIGH_CURRENT`
+* `<axis>.encoder.config.cpr = 4000`
+* `<axis>.encoder.config.calib_range = 0.05`
+* `<axis>.motor.config.calibration_current = 10.0`
+* `<axis>.motor.config.resistance_calib_max_voltage = 12.0`
+* `<axis>.controller.config.vel_limit = 50000`
+
+Lots of other values can get you. It's a process. Thankfully there is a lot of good people that will help you debug calibration problems. 
+
+If calibration works, congratulations.
+
+Now try: 
+* `<axis>.requested_state = AXIS_STATE_CLOSED_LOOP_CONTROL`
+* `<axis>.controller.set_vel_setpoint(3000,0) `
+let it loop a few times and then set:
+* `<axis>.requested_state = AXIS_STATE_IDLE`
+
+Do you still have no errors? Awesome. Now set these variables:
+* `<axis>.encoder.config.pre_calibrated = True`
+* `<axis>.motor.config.pre_calibrated  = True `
+
+And see if ODrive agrees that calibration worked by just running
+* `<axis>.encoder.config.pre_calibrated`
+
+(using no "= True" ). Make sure that 'pre_calibrated' is in fact True. 
+
+Also, if you have calibrated and encoder.pre_calibrated is equal to true, and you had no errors so far. Run this: 
+* `<axis>.encoder.config.use_index = True`
+* `odrv0.save_configuration()`
+* `odrv0.reboot()`
+
+and now see if after a reboot you can run: 
+* `<axis>.requested_state = AXIS_STATE_ENCODER_INDEX_SEARCH`
+
+without getting errors. 
+
+## Encoder Problems
```

Comment:
This section seems kinda strange overall, maybe you want to pull in encoder noise as a sebsection?

Reply:
I agree. I have changed the language. Instead of:
## Encoder problems. 

I changed to: 
"## What happens if calibration fails
There are subtle ways that encoder problems will impact your ODrive. For example, ODrive may not complete the calibrate sequence when you go to:"